modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -126,11 +128,13 @@ Java equivalents, such as `America/Los_Angeles` and
 
 == Debugging
 
-In the Studio *Debug Mode* view, the Mule app disables the triggering of events. You can
-instead execute the Scheduler component when debugging by clicking the green arrow icon
+In the Studio *Debug Mode* view, the Mule app disables triggering of Schedulers. When debugging, you can
```

Comment:
```suggestion
In the Studio *Debug Mode* view, a Mule application disables triggers on Scheduler components. When debugging, you can
```

Reply:
We need to verify whether it's true that Studio disables triggers when in debug mode.

Reply:
It does not disable the Scheduler

Reply:
Yep, I'm finding that it only stops if you stop the flow by adding a break point or if you actively select Stop Scheduler from the right-click menu on the Scheduler...